EssentialCSharp.Web/Views/Shared/_Layout.cshtml

```diff
@@ -7,7 +7,7 @@
 <head>
     @{
         const string imageUrl = "https://essentialcsharp.com/images/icon.png";
-        const string description = "Welcome to one of the most venerable and trusted franchises you could dream of in the world of C# books and probably far beyond! -Mads Torgersen, C# Lead Designer, Microsoft";
+        const string description = "Accelerate your development knowledge with C# expert Mark Michaelis� free, online comprehensive C# tutorial and reference that is updated through C# 11.0";
```

Comment:
Is this the right character?
![image](https://github.com/IntelliTect/EssentialCSharp.Web/assets/952248/a53b0507-cab9-44b1-a3d2-f46f5f623cdf)

Reply:
Hm, weird... It should be a ' and it shows right on github. Will change?
![image](https://github.com/IntelliTect/EssentialCSharp.Web/assets/22186029/5c027435-ced1-4923-ad31-c1139a6a809a)